code-challenges.test.js

```diff
@@ -65,5 +114,28 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("vowelSniper", () => {
+  it("creates a function that takes in a string and logs the index of the first vowel", () => {
+    expect(vowelSniper(vowelTester1)).toEqual(1)
+    expect(vowelSniper(vowelTester2)).toEqual(0)
+    expect(vowelSniper(vowelTester3)).toEqual(2)
+  })
+})
+//RED ACQUIRED
 
 // b) Create the function that makes the test pass.
+
+//PSEUDOCODE
+//Create a function called vowelSniper that passes a parameter of string
+//Create a for loop to iterate through the length of the string
+//Make a conditional occur at every letter that uses the logical OR operator for every a, e, i, o, and u.
+//If the value returns true for any of the vowels, return the current index
+
+const vowelSniper = (string) => {
```

Comment:
Now that you have a good handle on the for loop. Consider incorporating a high order function